runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -554,12 +556,15 @@ public void onScriptCallbackEvent(ScriptCallbackEvent scriptCallbackEvent)
 				clientThread.invokeLater(() -> confirmKickPlayer(kickPlayerName));
 				break;
 			}
-			case "clanChatChannelRebuild":
-				if (config.showIgnores())
-				{
-					clientThread.invoke(() -> colorIgnoredPlayers(config.showIgnoresColor()));
-				}
-				break;
+		}
+	}
+
+	@Subscribe
+	public void onScriptPostFired(ScriptPostFired event)
+	{
+		if (event.getScriptId() == CLAN_CHAT_CHANNEL_REBUILD && config.showIgnores())
+		{
+			clientThread.invoke(() -> colorIgnoredPlayers(config.showIgnoresColor()));
```

Comment:
Not sure why this was on the clientthread before, both these events are always on the client thread